test/powershell/Language/Operators/NullConditional.Tests.ps1

```diff
@@ -172,6 +188,16 @@ Describe 'NullCoalesceOperations' -Tags 'CI' {
         It 'Lhs is $?' {
             {$???$false} | Should -BeTrue
         }
+
+        It 'Should only evaluate LHS once when it is NOT null' {
+            (Set-TestStateWithStringResult) ?? 'Nothing' | Should -BeExactly 'Test'
+            $testState.Value | Should -Be 1
```

Comment:
The test function is used once and creates "side effect".
Please inline the function
```suggestion
            $testState = [pscustomobject]@{ Value = 0 }
            ( & { 'Test'; [void]$testState.Value++} ) ?? 'Nothing' | Should -BeExactly 'Test'
            $testState.Value | Should -Be 1
```